pkg/kubelet/dockershim/network/plugins.go

```diff
@@ -192,12 +192,12 @@ func (plugin *NoopNetworkPlugin) Init(host Host, hairpinMode kubeletconfig.Hairp
 	// it was built-in.
 	utilexec.New().Command("modprobe", "br-netfilter").CombinedOutput()
 	if err := plugin.Sysctl.SetSysctl(sysctlBridgeCallIPTables, 1); err != nil {
-		klog.Warningf("can't set sysctl %s: %v", sysctlBridgeCallIPTables, err)
+		klog.InfoS("can't set sysctl bridge-nf-call-iptables", "sysctlBridgeCallIPTables", sysctlBridgeCallIPTables, "err", err)
```

Comment:
No need to pass consts
```suggestion
		klog.InfoS("can't set sysctl bridge-nf-call-iptables", "err", err)
```